docker/Dockerfile.grpc-echo

```diff
@@ -0,0 +1,33 @@
+# Copyright 2024 The Kubernetes Authors.
```

Comment:
Is there any way we could reuse one of our existing echo servers for this? Maybe add to the one we already have?

Reply:
Done. I've incorporated it into echo-basic.